Refactor(eos_designs)!: Change the default false for wan_use_agent_env_var_for_kernel_software_forwarding_ecmp (#5959)
aristanetworks/ansible-avd: docs/porting-guides/6.x.x.md

+ underlay_ospf_graceful_restart: false
```

> ### For EOS kernel forwarding, ECMP programming is not enabled by default

Reviewer: This is not correct - both commands do the same thing. one is the old way (via ENV variable for the AGENT) the other one is the new way (via the proper CLI)

```suggestion
### Programming all forwarding paths of ECMP routes in the kernel is now enabled via the CLI for WAN routers
```

> In AVD 5.x the default value of `wan_use_agent_env_var_for_kernel_software_forwarding_ecmp` was `true`.

Reviewer: ```suggestion
In AVD 5.x, for historical reason, the programming was enabled by default using an environment variable for the Agent. Effectively, it meant that the default value of `wan_use_agent_env_var_for_kernel_software_forwarding_ecmp` was `true`.
```

> Starting with AVD 6.0.0, the default value of `wan_use_agent_env_var_for_kernel_software_forwarding_ecmp` has been changes to `false`.

Reviewer 2: ```suggestion
Starting with AVD 6.0.0, the default value of `wan_use_agent_env_var_for_kernel_software_forwarding_ecmp` has been changed to `false`.
```

> Starting with AVD 6.0.0, the default value of `wan_use_agent_env_var_for_kernel_software_forwarding_ecmp` has been changed to `false`.

Reviewer: ```suggestion
Starting with AVD 6.0.0, the default value of `wan_use_agent_env_var_for_kernel_software_forwarding_ecmp` has been changed to `false`. The functionality is now enabled by default via the proper CLI.
```

> To retain the previous behavior, the following change is required:

Reviewer: ```suggestion
To retain the previous behavior, for instance because your device does not support the CLI introduced in EOS 4.33.2F, the following change is required:
```
